wordpress-mobile/WordPress-iOS
# [Notifications Refresh] Replace tabs with dropdown in notifications screen

Fixes https://github.com/Automattic/wordpress-mobile/issues/9

| Light | Dark |
| ----- | ----- |
| ![CleanShot 2024-04-24 at 10 15 48@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/8bd78ee5-05da-4286-8c0d-89ae9c01ab19) |  ![CleanShot 2024-04-24 at 10 16 02@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/f4da509c-3431-4b06-bcfe-59e3ca6b30ea) |

## Test Instructions

1. Run the app and login.
2. Navigate to the Notifications screen
3. Click on the title in the navigation bar
4. It should display a dropdown menu
5. Click on the menu options
6. The dropdown should be dismissed, and then switch to the filter category you clicked, the title should be updated.

## Regression Notes
1. Potential unintended areas of impact
Make sure the correct filter is applied when selecting dropdown options.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:


## Testing checklist:
